src/families/near/transfer.js

```diff
@@ -69,6 +69,18 @@ export default class Transfer extends BaseTask {
             ),
             depo: new ArgsBig("1", "1", "1", "yocto")
         });
+        
+        this.loadErrors = (() => {
+
+            for (let e in this.baseErrors)
+                this.errors[e].validOrNull(this.call[e])
+
+            this.errors.receiver.validOrNull(this.call.args.value.receiver_id);
+            this.errors.amount.validOrNull(this.call.args.value.amount);
+
+            WALLET.then(() => this.updateFT());
+
+        }).bind(this)
```

Comment:
Do we still need this? I think it's from some old commit for fixing cloning bugs